README.md

```diff
@@ -127,6 +127,122 @@ When generating configuration, the program parses structured data (JSON) found i
 
 Similarly, the router watches its _own_ `routerConfig` annotations to dynamically construct global Nginx configuration.
 
+## Configuration Guide
+
+### Environment variables
+
+Router configuration is driven almost entirely by annotations on the router's replication controller and the services of all routable applications-- those labeled with `routable=true`.
+
+One exception to this, however, is that in order for the router to discover its own annotations, the router must be configured via environment variable with some knowledge of its own namespace.  (It cannot query the API for information about itself without knowing this.)
+
+The `NAMESPACE` environment variable is required by the router and it should be configured to match the Kubernetes namespace that the router is deployed into.  If no value is provided, the router will assume a value of `default`.
+
+For example, consider the following Kubernetes manifest.  Given a manifest containing the following metadata:
+
+```
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-router
+  namespace: deis
+# ...
+```
+
+The corresponding template must inject a `NAMESPACE=deis` environment variable into router containers as in this snippet from the same manifest:
+
+```
+# ...
+spec:
+  # ...
+  template:
+    # ...
+    spec:
+      containers:
+      - name: deis-router
+        # ...
+        env:
+        - name: NAMESPACE
+          value: deis
```

Comment:
Yup.  I actually have PR'ed that in this repo's manifests and it's already fixed in the charts, but overlooked it in the docs.  Nice catch.